.github/files/markdown.links.config.json

```diff
@@ -0,0 +1,28 @@
+{
+	"ignorePatterns": [
+		{
+			"pattern": "^http://127.0.0.1"
+		},
+		{
+			"pattern": "^http://localhost"
+		},
+		{
+			"pattern": "^https://img.shields.io/badge"
```

Comment:
why ignore shields? these should work in theory...

Reply:
Yeah. I don't know why it failed... I will remove them form ignore list again and see what happened.